prow/getting_started_deploy.md

```diff
@@ -261,6 +261,25 @@ label. When you make a change to the plugin config and push it with `make
 update-plugins`, you do not need to redeploy any of your cluster components.
 They will pick up the change within a few minutes.
 
+### Set namespaces for prowjobs and test pods
+
+Add the following to `config.yaml`:
+
+```yaml
+prowjob_namespace: default
+pod_namespace: test-pods
+```
+
+By doing so, we keep prowjobs in the `default` namespace and test pods in the
+`test_pods` namespace.
+
+You can also choose other names. Remember to update the RBAC roles and
+rolebindings afterwards.
+
+**Note**: If you set or update the `prowjob_namespace` or `pod_namespace`
+fields after deploying the prow components, you will need to redeploy them
+so that they pick up the change.
```

Comment:
I'm not sure if this is still true, but if we don't tell users they _must_ do this